records_mover/records/hints.py

```diff
@@ -35,7 +43,7 @@ def cant_handle_hint(fail_if_cant_handle_hint: bool, hint_name: str, hints: Reco
         raise NotImplementedError(f"Implement hint {hint_name}={repr(hints[hint_name])} " +
                                   "or try again with fail_if_cant_handle_hint=False")
 
-
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker